src/main/java/ai/zuva/api/ZdaiApiClient.java

```diff
@@ -0,0 +1,268 @@
+package ai.zuva.api;
+
+import ai.zuva.exception.ZdaiClientException;
+import ai.zuva.exception.ZdaiApiException;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import okhttp3.*;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+public class ZdaiApiClient {
```

Comment:
I'm wondering if we should actually call this DocAIApiClient.

Reply:
I'm happy with both. Going with `DocAIApiClient` makes it more product-focused since we have no reference to the company name (Z). So let's go with your suggestion of calling the class `DocAIApiClient`

Reply:
I called it DocAIClient (dropping `Api`, which doesn't really add much). Should I also move everything to `ai.zuva.docai`?